kodein/build.gradle

```diff
@@ -37,58 +21,13 @@ dependencies {
 
     testCompile "junit:junit:$junit_version"
 
-    testCompile project(':kodein-core').sourceSets.test.output
```

Comment:
It appears to be working without it. I saw that there were 2 other places that use the same statement (in kodein-conf/build.gradle and kodein-erased/build.gradle) and I tried removing it there as well, and everything still appears to be working. I'll leave this one commented out for now. Let me know if you want me to remove them.